components/petdex/Footer/Footer.tsx

```diff
@@ -0,0 +1,58 @@
+import Facebook from '@/components/petdex/Footer/assets/Facebook'
+import Instagram from './assets/Instragram'
+import LogoPetdex from './assets/LogoPetdex'
+import Twitter from './assets/Twitter'
+
+export default function Footer() {
+  return (
+    <footer className="flex w-full items-center justify-center bg-[#1E2833]">
+      <div className="w-full text-white">
+        <div className="grid grid-flow-row items-center justify-center gap-y-8 py-8 lg:grid-flow-col lg:gap-x-52">
+          <div className="min-w-50 flex justify-center lg:col-start-2">
+            <LogoPetdex />
+          </div>
+
+          <div className="flex min-w-52 items-center justify-center gap-10 text-sm">
+            <a href="/">
```

Comment:
Aqui como ta em next da pra usar a tag <Link>
```suggestion
            <Link href="/">
```